src/vs/workbench/contrib/terminal/browser/terminalGroup.ts

```diff
@@ -19,7 +19,7 @@ const enum Constants {
 	/**
 	 * The minimum size in pixels of a split pane.
 	 */
-	SplitPaneMinSize = 180,
+	SplitPaneMinSize = 80,
```

Comment:
Would we hit this on a smaller form factor like an ipad?

Reply:
I don't think so, the workbench minimum gets hit first